articles/azure-monitor/app/sharepoint.md

```diff
@@ -132,7 +134,8 @@ function onRequestFail(sender, args) {
 
 
 
-## Next Steps
+## Next steps
```

Comment:
```suggestion
## Next Steps
```

Reply:
Hi @xiao-lix Azure and Microsoft style is to use sentence case in headings and titles. The heading should remain _Next steps_ to be in alignment with this guidance. Thanks.

Reply:
I see, thanks. I was thinking that was me mistakenly making it lowercase. thanks.